packages/linked-data-proof/package.json

```diff
@@ -47,6 +47,7 @@
     "typescript": "^3.9.5"
   },
   "dependencies": {
+    "@transmute/did-key-ed25519": "^0.3.0-unstable.5",
```

Comment:
hmm is this really a "dependency" or a "dev dep"?

Reply:
I think I may actually be able to remove this now. And may only be dev dependency because it's only used in tests I believe if I can't remove it.

Reply:
This has been moved to dev dependency!